janitor/functions/pivot.py

```diff
@@ -70,7 +71,7 @@ def pivot_longer(
         0           5.1          3.5           1.4          0.2     setosa
         1           5.9          3.0           5.1          1.8  virginica
 
-        Replicate pandas' melt:
+    Replicate pandas' melt:
```

Comment:
Same here, the **entire** Examples block should be indented on the same level. So the indentation of your `Replicate pandas' melt` line is off.

```python
Examples:
    This is a text block.

     >>> x = 1

     This is another text block.
```

The way to understand this is, we denote new sub-blocks (paragraphs) within the Examples section with new lines, not with indentations.

I think there's no visual difference in the output (whether we indent "Replicate pandas' melt" text or not), just a difference in the parsed AST. so this is just a nitpick. :)